06-exceptions-mandatory-tasks/src/com/fsher/exercise4/exceptions/NumberZeroException.java

```diff
@@ -0,0 +1,7 @@
+package com.fsher.exercise4.exceptions;
+
+public class NumberZeroException extends Exception {
+    public NumberZeroException() {
+        super("Number should not be equal to zero");
```

Comment:
You may want your program to crash on it, so from my POV it's better to have this as a `RuntimeException`, so that you don't have to catch it. It's not a mistake, just my opinion